src/core/binding.c

```diff
@@ -382,6 +382,7 @@ _Success_(return != NULL)
 QUIC_LISTENER*
 QuicBindingGetListener(
     _In_ QUIC_BINDING* Binding,
+    _In_opt_ QUIC_CONNECTION* Connection,
```

Comment:
You're probably going to need to `UNREFERENCED_PARAMETER` for the no logging case.